.travis.yml

```diff
@@ -10,6 +11,12 @@ branches:
 before_install:
 - echo 'DOCKER_OPTS="-H unix:///var/run/docker.sock -s devicemapper"' | sudo tee /etc/default/docker
   > /dev/null
+- sudo apt-get install -qq pkg-config fuse
+- sudo modprobe fuse
+- sudo chmod 666 /dev/fuse
+- sudo chown root:$USER /etc/fuse.conf
```

Comment:
This was my own doing ... but I think this line is redundant given the lines below